app/services/idv/session.rb

```diff
@@ -180,7 +180,6 @@ def invalidate_in_person_pii_from_user!
         user_session['idv/in_person'][:pii_from_user] = nil
         # Mark the two FSM steps as incomplete so that they can be re-entered.
         user_session['idv/in_person'].delete('Idv::Steps::InPerson::StateIdStep')
-        user_session['idv/in_person'].delete('Idv::Steps::InPerson::AddressStep')
```

Comment:
Nit: Can we update comment- it is no longer two FSM steps- maybe something like.... "Mark the one FSM step as incomplete so that it cannot be re-entered" or "Mark the only FSM step as..." on line 181.  **Only if you edit anything else in here- else just ignore.**